index.js

```diff
@@ -243,11 +243,35 @@ var wordList = [
   "work","worker","world","worried","worry","worse","worth","would",
   "wrapped","write","writer","writing","written","wrong","wrote","yard",
   "year","yellow","yes","yesterday","yet","you","young","younger",
-  "your","yourself","youth","zero","zoo"
+  "your","yourself","youth","zero","zebra","zipper","zoo","zulu"
 ];
 
 function words(options) {
+
   function word() {
+    if (options && options.maxLength) {
+      return GenerateWordWithMaxLength();
+    }
+
+    else {
+      return GenerateRandomWord();
+    }
+  }
+
+  function GenerateWordWithMaxLength() {
```

Comment:
I really like how you structured this, but functions that are not constructors should never have a capitalized first letter in js.